ui-v2/tests/integration/mixins/with-resizing-test.js

```diff
@@ -0,0 +1,26 @@
+import { module } from 'qunit';
+import test from 'ember-sinon-qunit/test-support/test';
+import { setupTest } from 'ember-qunit';
+import EmberObject from '@ember/object';
+import Mixin from 'consul-ui/mixins/with-resizing';
+module('Integration | Mixin | with-resizing', function(hooks) {
+  setupTest(hooks);
+  test('window.addEventListener, resize and window.removeEventListener are called once each through the entire lifecycle', function(assert) {
+    const win = {
+      innerWidth: 0,
+      innerHeight: 0,
+      addEventListener: this.stub(),
+      removeEventListener: this.stub(),
```

Comment:
Neat. I need to look into `ember-sinon-qunit`. I'm just using plain `sinon` right now, but I love spies.